src/ProducerSideService/ProducerSideServer.cpp

```diff
@@ -20,6 +20,7 @@ bool ProducerSideServer::BuildAndStart(const std::string& uri) {
   ORBIT_CHECK(server_ == nullptr);
 
   grpc::ServerBuilder builder;
+  builder.SetMaxReceiveMessageSize(std::numeric_limits<int32_t>::max());
```

Comment:
Under the assumption from the other comment that the message that exceeds the limit is `CaptureOptions`, why does the receive message size in `ProducerSideServer` make a difference? `ProducerSideServer` sends `CaptureOptions`, it doesn't receive it. Or does the receive message size set on the server also affect the client?

Reply:
Removed, see comment above.